src/java/picard/analysis/artifacts/CollectSequencingArtifactMetrics.java

```diff
@@ -86,9 +86,6 @@
             "i.e. they're expected to face the same direction.")
     public boolean TANDEM_READS = false;
 
-    @Option(doc = "When available, use original quality scores for filtering.")
```

Comment:
instead of deleting default false and add comment in doc explaining the increase in run-time.